CONTRIBUTING.md

```diff
@@ -1,3 +1,380 @@
 # Contributing
 
-TODO
+These guidelines will help you get started with the Starboard project.
+
+## Table of Contents
+
+- [Contribution Workflow](#contribution-workflow)
+  - [Issues and Discussions](#issues-and-discussions)
+  - [Pull Requests](#pull-requests)
+- [Set up your Development Environment](#set-up-your-development-environment)
+- [Build Binaries](#build-binaries)
+- [Run Tests](#run-tests)
+  - [Run Unit Tests](#run-unit-tests)
+  - [Run Integration Tests](#run-integration-tests)
+  - [Cove Coverage](#code-coverage)
+- [Custom Resource Definitions](#custom-resource-definitions)
+  - [Generate Code](#generate-code)
+- [Test Starboard Operator](#test-starboard-operator)
+  - [Prerequisites](#prerequisites)
+  - [In Cluster](#in-cluster)
+  - [Out of Cluster](#out-of-cluster)
+  - [Enable Aqua Scanner](#enable-aqua-scanner)
+- [Operator Lifecycle Manager (OLM)](#operator-lifecycle-manager-olm)
+  - [Install OLM and Operator Marketplace](#install-olm-and-operator-marketplace)
+  - [Publish the OLM Bundle to Quay.io](#publish-the-olm-bundle-to-quayio)
```

Comment:
There's a way to add the OLM bundle to a Docker container and use that image reference instead of publishing it to Quay.io. However, I need to test such approach and then update this docs.